src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -184,11 +135,6 @@ public new CSharpParseOptions WithDocumentationMode(DocumentationMode documentat
                 return this;
             }
 
-            if (!documentationMode.IsValid())
-            {
-                throw new ArgumentOutOfRangeException(nameof(documentationMode));
-            }
-
             return new CSharpParseOptions(this) { DocumentationMode = documentationMode };
```

Comment:
Same.

---
In reply to: [101868868](https://github.com/dotnet/roslyn/pull/17113#discussion_r101868868) [](ancestors = 101868868)